dor/providers/translocator.py

```diff
@@ -37,16 +37,12 @@ def object_data_directory(self) -> Path:
         if self.root_identifier is None:
             raise Exception()
         return self.package_directory() / "data" / self.root_identifier
-    
-    def _apply_relative_path(self, path: Path, path_to_apply: Path) -> Path:
-        return (path / path_to_apply).resolve().relative_to(self.object_data_directory().resolve())
 
     def get_bundle(self, entries: list[Path]) -> Bundle:
         resolved_entries = []
         for entry in entries:
             resolved_entries.append(
-                self._apply_relative_path(self.object_data_directory() / "descriptor", entry)
-            )
+                str(entry))
```

Comment:
We can probably just remove this accumulation all together now, right? Looks like `entries` can be passed directly to `Bundle`